torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py

```diff
@@ -106,8 +106,7 @@ def _forward_call(self, *inputs, **kwargs):
 
         # Use IO binding
         onnx_input_names = [inp.name for inp in self._onnx_models.exported_model.graph.input]
-        input_info = _io.parse_inputs_for_onnx_export(self._module_parameters, None, schema, inputs, kwargs)
-        inputs = _utils_infer.get_user_inputs(onnx_input_names, input_info, inputs, kwargs, self._device)
+        inputs = _utils_infer.get_user_inputs(onnx_input_names, self._flattened_module._input_info, inputs, kwargs, self._device)
```

Comment:
Where are we setting `self._flattened_module._input_info`?

Reply:
ok, I see it is being set in `_export_model`. Nevermind. Thanks for making this change.